internal/storage/bank_account_repository.go

```diff
@@ -19,14 +19,14 @@ func NewBankAccountRepository(db *gorm.DB) BankAccountRepository {
 
 // All ...
 func (p *BankAccountRepository) All() ([]model.BankAccount, error) {
-	bankAccounts := []model.BankAccount{}
+	var bankAccounts []model.BankAccount
```

Comment:
If you do this change, when there is no record on database, the response json object will be null, instead if empty array which is not a good usecase on frontend. Can you revert this change

Reply:
I guess, I have tested it and it was working pretty ok. I did not manage to get null , from any of them. I have tested following endpoint without  having any bank account.

`http://localhost:3625/api/bank-accounts`

if it is possible, could you give me PoC example ? for that scenario which I can understand the problem better. When you have time, of course.